processing/src/test/java/io/druid/query/groupby/GroupByQueryRunnerTest.java

```diff
@@ -8743,7 +8768,10 @@ public void testMergeResultsWithLimitPushDownSortByAgg()
         GroupByQueryRunnerTestHelper.createExpectedRow("2011-04-02", "alias", "travel", "rows", 2L, "idx", 243L)
     );
 
-    Iterable<Row> results = Sequences.toList(mergedRunner.run(allGranQuery, context), Lists.<Row>newArrayList());
+    Iterable<Row> results = Sequences.toList(
+        mergedRunner.run(QueryPlus.wrap(allGranQuery), context),
+        new ArrayList<Row>()
```

Comment:
There is official preference. I suggested to standardize on `new ArrayList<>()` form before, but it was not supported: https://github.com/druid-io/druid/pull/3676#discussion_r87468409